### PR TITLE
add jobs of labels ocm-resources, day1-negative, day1-supplemental

### PR DESCRIPTION
**Pull request by yuwang-RH**
add jobs of labels ocm-resources, day1-negative, day1-supplemental

### CODE REVIEW

**Comment by yuwang-RH** [comment]
@xueli181114 Could you help review this?

**Comment by yuwang-RH** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-negative-f7

**Comment by yuwang-RH** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-supplemental-f3

**Comment by yuwang-RH** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-supplemental-f3

**Comment by yuwang-RH** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-ocm-resources-f3

**Comment by yuwang-RH** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-negative-f7

File: ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml [hunk 1]
```diff
@@ -129,6 +129,39 @@ tests:
...
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: stable
+      TEST_LABEL_FILTERS: (ocm-resources)&&!Exclude
+      TEST_PROFILE: "null"
+    test:
+    - ref: rosa-tests
+    - ref: rosa-tests-report
+- as: rosa-negative-f7
```
Comment (xueli181114) [inline ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml:143]: name it with `rosa-day1-negative-f7`

File: ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml [hunk 2]
```diff
@@ -129,6 +129,39 @@ tests:
...
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: stable
+      TEST_LABEL_FILTERS: (day1-negative)&&!Exclude
+      TEST_PROFILE: "null"
+    test:
+    - ref: rosa-tests
+    - ref: rosa-tests-report
+- as: rosa-supplemental-f3
```
Comment (xueli181114) [inline ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml:154]:
```suggestion
- as: rosa-day1-supplemental-f3
```

File: ci-operator/step-registry/rosa/tests/rosa-tests-commands.sh [hunk 3]
```diff
@@ -9,9 +9,11 @@ export TEST_PROFILE=${TEST_PROFILE:-}
 TEST_LABEL_FILTERS=${TEST_LABEL_FILTERS:-}
 TEST_TIMEOUT=${TEST_TIMEOUT:-"4h"}
 
-CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
-echo "Working on the cluster: $CLUSTER_ID"
-export CLUSTER_ID # maybe we should get cluster_id by TEST_PROFILE
+if [[ "$TEST_PROFILE" != "null" ]]; then
+  CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
+  echo "Working on the cluster: $CLUSTER_ID"
+  export CLUSTER_ID # maybe we should get cluster_id by TEST_PROFILE
```
Comment (xueli181114) [inline ci-operator/step-registry/rosa/tests/rosa-tests-commands.sh:15]:
```suggestion
  export CLUSTER_ID # This is already enhanced in rosa repo. And will remove this in another PR
```

**Comment by yuwang-RH** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-negative-f7

**Comment by yuwang-RH** [comment]
@ciaranRoche Could you help review then approve this PR?Thanks.

**Review by ciaranRoche — COMMENTED** [review]
/lgtm

**Pull request merged**